core/src/main/scala/filodb.core/downsample/DownsampledTimeSeriesShard.scala

```diff
@@ -216,17 +216,20 @@ class DownsampledTimeSeriesShard(rawDatasetRef: DatasetRef,
       case FilteredPartitionScan(split, filters) =>
 
         if (filters.nonEmpty) {
-          val res = partKeyIndex.partIdsFromFilters(filters,
+
+          val res = partKeyIndex.partKeyRecordsFromFilters(filters,
```

Comment:
Perhaps rename this to `records`?  The moving of res threw me off down there as it wasn't clear that it wasn't the same thing.